addons/controllers/kapp-controller/kappcontrollerconfig_controller.go

```diff
@@ -0,0 +1,219 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package controllers implements k8s controller functionality for kapp-controller config CRD.
+package controllers
+
+import (
+	"context"
+	"fmt"
+
```

Comment:
order imports into 3 groups (std lib, external, local github.com/vmware-tanzu) here and other places